src/utils/MultiInviter.ts

```diff
@@ -170,6 +172,27 @@ export default class MultiInviter {
                     errcode: USER_ALREADY_INVITED,
                     error: "Member already invited",
                 });
+            } else if (member?.membership === "ban") {
```

Comment:
slight concern with checking before attempting the invite is it is vulnerable to Element's state drifting out of sync with the server.  We could refuse to invite someone who isn't actually banned, etc.  It might have been nicer to prompt to unban if the initial attempt to invite fails.

But I guess we already have this problem with the other checks here, so :man_shrugging: